File: typescript/packages/management/src/tools/list_triggers.ts
import type { ToolContext, ToolHandler } from '../registrar.js';

export const listTriggers: ToolHandler = async (
  ctx: ToolContext,
  input: Record<string, unknown>,
) => {
  const slug = String(input.agent_slug);
  return ctx.client.agents.triggers.list(ctx.sandboxId, slug);
};

File: python/copass-management/src/copass_management/tools/list_triggers.py
from __future__ import annotations

from typing import Any, Dict, TYPE_CHECKING

if TYPE_CHECKING:
    from copass_management.registrar import ToolContext


async def list_triggers(ctx: "ToolContext", input: Dict[str, Any]) -> Any:
    return await ctx.client.agents.triggers.list(
        ctx.sandbox_id, str(input["agent_slug"])
    )
